example.env

```diff
@@ -0,0 +1,17 @@
+# Reference: https://github.com/frappe/frappe_docker/blob/main/docs/images-and-compose-files.md
+
+FRAPPE_VERSION=v13.16.0
+DB_PASSWORD=123
+
+# Only with ERPNext override
+ERPNEXT_VERSION=v13.16.1
+
+# Only with HTTPS override
+LETSENCRYPT_EMAIL=mail@example.com
+
+# This environment variable is not required. Default value is `$$host` which resolves site by host.
+# For example, if your host is `example.com`, site's name should be `example.com`,
+# or if host is `127.0.0.1` (local debugging), it should be `127.0.0.1` This variable allows
+# to override described behavior. Let's say you create site named `mysite`
+# and do want to access it by `127.0.0.1` host. Than you would set this variable to `mysite`.
+FRAPPE_SITE_NAME_HEADER=
```

Comment:
This restricts the bench to one site?

Reply:
Only if the variable set. I saved current behavior. Default value of this variable set in `compose.yaml` to `$$host` (escaped from `$host`).